src/components/attributeSelection/optionsSelection.ts

```diff
@@ -8,11 +8,17 @@ export default class OptionsSelection extends Vue {
   public module!: IAttributeModuleWithOptions;
   @Prop({ required: false, type: Boolean })
   public twoColumns!: boolean;
+  public showPhosphorusMessage: boolean = false;
+
+  get isWaterQualityModule() {
+    return this.module.name === '$waterQuality';
+  }
 
   get selectedIds() {
     return this.module.selectedIds;
   }
   set selectedIds(e) {
+    this.includesPhosphorus(e) ? this.showPhosphorusMessage = true : this.showPhosphorusMessage = false;
```

Comment:
Tän vois kääntää muotoon `this.showPhosphorusMessage = this.includesPhosphorus(e)`